### PR TITLE
Fix merge issues

### PR DESCRIPTION
**Pull request by JNikolo**
This pull request attempts to solve a few issues that happened after **merge #14** . So far I did:
- I deleted `express` dependency and added `prisma` as dev-dependency for the `server` application. 
- Merged code from `landing-screen-page` that was lost during the merge previously mentioned in the `client` app

What needs to be done:
- Test Expo app to make sure it's running or fix problems
- Test the fastify app or fix problems

### CODE REVIEW

**Review by AnimeshDeb — APPROVED** [review]
Thank you for addressing the issues that arose after merge #14, especially with removing the Express dependency and merging the lost landing-screen-page code—greatly appreciated! I’ve also made the necessary changes to my local files to ensure we avoid similar issues moving forward. Your effort in tackling these post-merge challenges is commendable—great work!

**Review by Chromium99 — APPROVED** [review]
File changes look good. Great progress!

**Review by Imtiaz5 — APPROVED** [review]

File: client/.gitignore
Comment (DeveloperMindset123) [inline client/.gitignore:1]: I believe .env was originally added to .gitignore, and I also removed the yarn.lock file, strange, it appears the changes didn't take place.

File: client/apps/expo/app/components/landingPageComponents/RenderImage.tsx
Comment (DeveloperMindset123) [inline client/apps/expo/app/components/landingPageComponents/RenderImage.tsx:1]: Right, I forgot to mention this added component earlier, thank you for adding this in

**Review by DeveloperMindset123 — APPROVED** [review]
Looks good, I believe the changes weren't originally reflected because I must have forgotten to merge my branch with main

**Review by wong-ja — COMMENTED** [review]
Thanks for adding the components folder and sorting out the merge issues. I appreciate your quick response to this matter. Good work!

**Pull request merged**